bika/lims/catalog.py

```diff
@@ -484,11 +483,7 @@ def _merge_both_catalogs(catalogs_definition, catalog_extensions):
                     logger.error(traceback.format_exc())
             else:
                 # If catalog id is not found in the original catalog
-                # definition, definition, rise an info and create the new dict
-                logger.info(
-                    "Catalog %s doesn't exist in Bika LIMS catalog "
-                    "definitions dictionary. A new catalog definition"
-                    " might be created." % ext_cat_id)
+                # definition, definition
```

Comment:
@Espurna need more clarity in this comment, something like: "If this catalog ID is not found in the catalogs definition, then ...........".